airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -316,6 +318,48 @@ public JdbcDatabase createDatabase(final JsonNode config) throws SQLException {
     return database;
   }
 
+  /**
+   * Retrieves connection_properties from config and also validates if custom
+   * jdbc_url parameters overlap with the default properties
+   *
+   * @param config A configuration used to check Jdbc connection
+   * @return A mapping of connection properties
+   */
+  protected Map<String, String> getConnectionProperties(final JsonNode config) {
+    final Map<String, String> customProperties = JdbcUtils.parseJdbcParameters(config, JdbcUtils.JDBC_URL_PARAMS_KEY);
+    final Map<String, String> defaultProperties = getDefaultConnectionProperties(config);
```

Comment:
Yes, the reason I chose to have `config` as opposed to `jdbcConfig` is because it will match the same arguments that `AbstractJdbcDestination#getDefaultConnectionProperties` gets passed from `AbstractJdbcDestination#getDataSource`. Also `jdbcConfig` is merely a subset of `config` so it will have the same information overall